### PR TITLE
feat: add object rotated event listener

### PR DESCRIPTION
**Pull request by jinwoo-kim-nhn**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has description for the breaking change

#### issue

* https://github.com/nhn/tui.image-editor/issues/285
* 사용자가 참조할수 있는 object 회전 이벤트 리스터가 없음

#### 작업내용
* objectRotated 이벤트 리스너 추가

### CODE REVIEW

File: src/js/imageEditor.js [hunk 1]
```diff
@@ -274,19 +291,20 @@ class ImageEditor {
...
-            'addText': this._handlers.addText,
-            'addObject': this._handlers.addObject,
-            'textEditing': this._handlers.textEditing,
-            'textChanged': this._handlers.textChanged,
-            'iconCreateResize': this._handlers.iconCreateResize,
-            'iconCreateEnd': this._handlers.iconCreateEnd,
-            'selectionCleared': this._handlers.selectionCleared,
-            'selectionCreated': this._handlers.selectionCreated,
-            'addObjectAfter': this._handlers.addObjectAfter
+            [MOUSE_DOWN]: this._handlers.mousedown,
```
Comment (dongsik-yoo) [inline src/js/imageEditor.js]: 오 깔끔하네요

File: src/js/imageEditor.js [hunk 2]
```diff
@@ -476,6 +494,25 @@ class ImageEditor {
...
     }
 
+    /**
+     * 'objectRotated' event handler
+     * @param {ObjectProps} props - object properties
+     * @private
+     */
+    _onObjectRotated(props) {
+        /**
+         * The event when scale factor is changed
```
Comment (dongsik-yoo) [inline src/js/imageEditor.js]: 주석 복붙 ㅎㅎ `scale factor`

File: test/imageEditor.spec.js [hunk 3]
```diff
@@ -74,6 +74,19 @@ describe('ImageEditor', () => {
...
+            const canvas = imageEditor._graphics.getCanvas();
+            const obj = new fabric.Object({});
+            const mock = {target: obj};
+            canvas.add(obj);
+
+            spyOn(imageEditor, 'fire').and.callThrough();
+
+            canvas.fire('object:rotating', mock);
+
+            expect(imageEditor.fire.calls.mostRecent().args[0]).toBe('objectRotated');
```
Comment (dongsik-yoo) [inline test/imageEditor.spec.js]: 테스트 코드에도 `OBJECT_ROTATED` 쓰면 되겠어요.

File: src/js/imageEditor.js (hunk 1 above)
Comment (jinwoo-kim-nhn) [reply src/js/imageEditor.js]: 감사합니다. !!

File: src/js/imageEditor.js (hunk 2 above)
Comment (jinwoo-kim-nhn) [reply src/js/imageEditor.js]: 복붙 걸렸네요 ;; 수정하였습니다. ㅎㅎ

File: test/imageEditor.spec.js (hunk 3 above)
Comment (jinwoo-kim-nhn) [reply test/imageEditor.spec.js]: 반영하였습니다. 감사합니다.

**Pull request merged**